.binstar.yml

```diff
@@ -6,7 +6,6 @@ platform:
   - win-64
   - win-32
 engine:
-  - python=2.6
```

Comment:
I guess so.  It would be straightforward to support 2.6.  It was easier to drop it.  What's your preference?